cli/src/cli.rs

```diff
@@ -137,12 +136,27 @@ pub enum Commands {
     },
 
     // ----- Information Commands ----- //
-    /// Print the information about the Git server that this node is hosting.
-    Git,
     /// Show the overall information of the given commit.
     Show { revision: String },
     /// Show the current status of the p2p network.
     Network,
+    /// Show the status of the Simperby repository.
+    ///
+    /// It checkes the following for the current directory:
+    ///
+    /// 1. Is it a valid Git repository?
+    /// 2. Does it contain a valid `.simperby/` directory?
+    /// 3. Does it have a valid `reserved/` directory?
+    /// 4. Does it have all the reserved branches and tags?
```

Comment:
I think it might be tricky for overlapping use of "reserved".

Reply:
Updated to `protected`